.changeset/muddy-migrations-melt.md

```diff
@@ -0,0 +1,10 @@
+---
+"@biomejs/biome": minor
+---
+
+Style runs that were enabled for users coming from Biome 1.x are no longer
```

Comment:
"Style runs" isn't something that users are aware of. Users only know about `biome migrate`. What about this, as alternative?

```md
`biome migrate` no longer enables style rules that were recommended in v1.
```